api/fast.py

```diff
@@ -6,7 +6,7 @@
 from fastapi import FastAPI, File, UploadFile, Form
 from fastapi.middleware.cors import CORSMiddleware
 
-from cookit.predict import Predictor
+from cookit.predict_lite import Predictor
```

Comment:
To be discussed if we want to deploy this model? If not the import should be changed to cookit.predict again